chapters/remote/remote-flows.md

```diff
@@ -0,0 +1,71 @@
+# Remote Flows with JinaD
+
+### Prerequisites
+
+Before you start, make sure you have read the [prerequisites for using JinaD](https://docs.jina.ai/chapters/remote/jinad.html#prerequisites).
+
+## REST API
+
+[Refer the detailed API reference for JinaD](https://api.jina.ai/daemon/#tag/flows)
+
+A common use case is running the Flow on the remote with pods distributed across different machines.
```

Comment:
on the remote machines? not sure what we are trying to say here

Reply:
This means, Flow is remote to the user & Pods are remote to the Flow. I don't know any better ways to improve this for now :) We'll change it later.